food/recipes/templates/recipes/index.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <title>K+A Recipes</title>
+  </head>
+  <body>
+    <h1>Recipes</h1>
+
+    {%for recipe in recipes_list%}
+    <a href="{% url 'recipes:detail' recipe.id %}">
+      <div class="card" style="width: 18rem;">
+        <img src="{{recipe.image_link}}" class="card-img-top" alt="A picture of {{recipe.food_name}}">
+        <div class="card-body">
+          <h5 class="card-title">{{recipe.name}}</h5>
+          <h6 class="card-text">{{recipe.convert_genres}}</h6>
+
+        </div>
+      </div>
+    </a>
+    {%endfor%}
+
+
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
```

Comment:
We don't actually need the JQuery and Popper libraries yet, try to comment them out and see if we can run without them.